base_rest/__manifest__.py

```diff
@@ -18,7 +18,7 @@
         "views/base_rest_view.xml",
     ],
     "assets": {
-        "web.assets_backend": [
+        "web.assets_common": [
```

Comment:
Looking again at this... IMO it's pointless and not recommendable to load these resources for ALL assets.

We should load them directly in the template or add our own assets.

Reply:
We can load lib assets directly in the template. It works the same. See my last fix

Reply:
@simahawk can you pls take a look at the last commit?

Reply:
much better!